docs/src/docs/usage/install/index.mdx

```diff
@@ -151,6 +151,29 @@ As such, we recommend using the fixed minor version and fixed or the latest patc
 For example, in our [GitHub Action](https://github.com/golangci/golangci-lint-action) we require users to explicitly set the minor version of `golangci-lint`
 and we always use the latest patch version.
 
+## Linter Deprecation Cycle
+
+A linter can be deprecated for various reasons, e.g. the linter stops working with a newer version of Go or the author has abandoned its linter.
+
+The deprecation of a linter will follow 3 phases:
```

Comment:
I propose to write these few sentences without `will` and highlight the most important phrases, like the following:

```
The deprecation of a linter follows three phases:

1. **Display of a warning message**: The linter can still be used (unless it's completely non-functional), but it's recommended to remove it from your configuration.
2. **Display of an error message**: At this point, you should remove the linter. The original implementation is replaced by a placeholder that does nothing.
3. **Removal of the linter** from golangci-lint.
```